react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -907,6 +909,12 @@ class Thumbnail extends Component<Props, State> {
             return this._renderFakeParticipant();
         }
 
+        if (isFakeScreenShareParticipant) {
+            return (<FakeScreenShareParticipant
```

Comment:
Still debating if we should create a separate component for the fake screens share participant or just modify _renderParticipant. 

If we create a new component, we can have a simpler version of a participant thumbnail with features specific to screen share. However, there will be a lot of code duplications such as onclick/mouse events and the top/bottom indicators. We will also need to maintain two similar yet different thumbnails

Reply:
From the rendering perspective, it will be harder to calculate thumbnail sizes if we introduce a new double tile. Are all indicators needed in the screen tile?

Reply:
If the tiles will be separate this will complicate the case when there's screen, but no camera. If we go with 2 tiles then the easiest will be to always render two. The main one will hold all of the indicators and in case there's only screen it would show the avatar as usual in this case. The screen one would have no indicators. @jallamsetty1 can you ask about this case?

Reply:
The plan was to go with 2 tiles always when SS track is added. If the camera is muted, avatar should be shown on the main one. We do not have to duplicate all the indicators on the fake screenshare thumbnail. The remote video menu on the screenshare thumbnail is not needed and also in the connection quality, only the resolution and frame rate needs to be displayed.